app_feup/lib/view/Pages/HomePageView.dart

```diff
@@ -26,10 +31,9 @@ class HomePageView extends StatelessWidget {
         shrinkWrap: false,
         padding: const EdgeInsets.all(20.0),
         children: <Widget>[
-          new Text(
-            'Favorites:',
-            style: Theme.of(context).textTheme.title,
-          ),
+          new GenericCard(
+            title: "Exames"
+            , child: new ExamCard(firstExam: exam, secondExam: exam,))
```

Comment:
Instead of creating a dummy exam here, the HomePage should call "getExams()" on start, which should place all of the exams' information on the AppState (with redux). Then, here in line 36, we would just feed the exam list from the AppState to the exam card. This way, things are more abstract in the "view" side of the app, allowing us to more easily maintain the code in the future.

Reply:
Exams are now loading from redux